<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="VideoTable" name="mainTable">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CRF</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Format</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Copy Meta</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New SIze</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Full Path</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="outputDisplay"/>
    </item>
    <item>
     <widget class="QWidget" name="controlWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/icons/plus.png</normaloff>:/icons/icons/plus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/icons/minus.png</normaloff>:/icons/icons/minus.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="compressButton">
           <property name="text">
            <string>Compress</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/icons/data-compression.png</normaloff>:/icons/icons/data-compression.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="compressOptGroup">
           <property name="title">
            <string>Compression Options</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="CRFLabel">
              <property name="text">
               <string>Constant Rate Factor</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="crfEdit">
              <property name="maximum">
               <number>51</number>
              </property>
              <property name="value">
               <number>23</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="formatLabel">
              <property name="text">
               <string>Format</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="formatEdit">
              <item>
               <property name="text">
                <string>H.264</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>H.265</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="copyMetaLabel">
              <property name="text">
               <string>Copy Metadata</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="copyMetaCheckbox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="setOptionsButton">
             <property name="text">
              <string>Set Options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="sameForAllCheckBox">
             <property name="text">
              <string>Same for All</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="OutputFolderGroup">
           <property name="title">
            <string>Output Folder</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="outputDirEdit"/>
              </item>
              <item>
               <widget class="QToolButton" name="outputDirBrowse">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="suffixLabel">
                <property name="text">
                 <string>Suffix:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="suffixEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAdd_Pictures"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Pictures"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionCompress"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionAdd_Pictures">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/open-folder.png</normaloff>:/icons/icons/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Add Pictures</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/logout.png</normaloff>:/icons/icons/logout.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/sweeping.png</normaloff>:/icons/icons/sweeping.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionCompress">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/data-compression.png</normaloff>:/icons/icons/data-compression.png</iconset>
   </property>
   <property name="text">
    <string>Compress</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoTable</class>
   <extends>QTableWidget</extends>
   <header>videotable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
